node/src/components/consensus/protocols/highway/round_success_meter.rs

```diff
@@ -52,8 +60,10 @@ impl<C: Context> RoundSuccessMeter<C> {
     fn check_proposals_success(&self, state: &State<C>, proposal_h: &C::Hash) -> bool {
         let total_w = state.total_weight();
 
-        let finality_detector =
-            FinalityDetector::<C>::new(max(total_w / 100 * THRESHOLD, Weight(1)));
+        let finality_detector = FinalityDetector::<C>::new(max(
+            total_w / *self.config.acceleration_ftt.denom() * *self.config.acceleration_ftt.numer(),
```

Comment:
I'd rather multiply before dividing, to make sure this isn't 0.
To avoid overflows, we'd have to convert to `u128` and back.